src/bsoncxx/document/value.cpp

```diff
@@ -53,11 +54,7 @@ value& value::operator=(const value& rhs) {
 }
 
 value::unique_ptr_type value::release() {
-    auto x = std::move(_data);
-
-    _data.release();
-
-    return x;
+    return std::move(_data);
```

Comment:
Per discussion, is this std::move required? I can't remember which way we decided it.